Makefile.in

```diff
@@ -95,6 +96,7 @@ else
 	BINMODESIGHUP:=4710
 endif
 
+export
```

Comment:
It should actually just be `export LIBS` for the tests' sub-makefile. But, it can be removed if LIBS is explicitly set in the sub-makefile. What's better for you?

Reply:
probably simpler to define `LIBS` in the sub-makefile, with the `./configure` script setting its value.